app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
+    if len(command_parts) != 3 or command_parts[0] != "mv":
+        raise ValueError("Invalid command format. "
+                         "Use 'mv source destination'.")
+
+    source_path = command_parts[1]
+    destination_path = command_parts[2]
+
+    is_directory = destination_path.endswith("/")
+
+    if not os.path.exists(source_path):
+        raise FileNotFoundError(f"'{source_path}' not found.")
+
+    if is_directory:
```

Comment:
But in this case when path will be:
dir1/file.txt
Your code will not create dir1